app/templates/partials/guidance.html

```diff
@@ -0,0 +1,17 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% for item in guidance_content.content %}
+  {%- if item.title -%}
+    <h3 class="venus">{{item.title}}</h3>
+  {% endif %}
+  {%- if item.description -%}
+    <h2 class="mars">{{item.description|safe}}</h2>
```

Comment:
https://github.com/ONSdigital/eq-survey-runner/pull/1133 hasn't merged yet so this is still required